charts/karpenter-crd/templates/karpenter.azure.com_aksnodeclasses.yaml

```diff
@@ -20,7 +20,18 @@ spec:
     singular: aksnodeclass
   scope: Cluster
   versions:
-  - deprecated: true
+  - additionalPrinterColumns:
+    - jsonPath: .status.conditions[?(@.type=='Ready')].status
+      name: Ready
+      type: string
+    - jsonPath: .metadata.creationTimestamp
```

Comment:
(fair enough w.r.t. some of the info only being in wide).

Wondering if we have this backwards btw?

Default:
- Ready
- ImageFamily

Wide:
- CreationTime
- (TBD other stuff?)

Reply:
The way it is right now (Age & Ready by default, ImageFamily in wide) seems right to me